jep/307/README.adoc

```diff
@@ -306,6 +324,118 @@ rather than critical to the operation of the update lifecycle. This will be
 used at a future point in time to better understand the runtime environments
 for the Jenkins and evergreen-client processes.
 
+
+=== Adding new Update Levels
+
+In order for client to receive new <<update-levels>>, an automated backend
+process should generating an `ingest.yaml` to be sent to the Evergreen backend
+service layer.
+
+==== `ingest.yaml`
+
+The `ingest.yaml` file should be machine-generated from the `essentials.yaml`
+with URLs and checksums for artifacts at specific point in time.  This provides
+the raw data which the Update service should use to create
+<<update-levels>>. The file should be checked into source control and managed
+via pull requests and automated changes to allow for thorough testing of the
+set time-based snapshot of artifacts.
+
+.ingest.yaml
```

Comment:
Should this file contain a `version: 1` tag to accomodate future changes?

For instance, in my craziest dreams, I would like us to optimize product distribution in the future by making downloads diff-friendly. This is obviously way too early, but e.g. `pack200` would be something I think would be very interesting to study to possibly reduce the download sizes by orders of magnitudes.